.travis.yml

```diff
@@ -62,14 +68,23 @@ script:
   - curl https://raw.githubusercontent.com/jhipster/generator-jhipster/master/test-integration/samples/.jhipster/Label.json -o .jhipster/Label.json
   - curl https://raw.githubusercontent.com/jhipster/generator-jhipster/master/test-integration/samples/.jhipster/Operation.json -o .jhipster/Operation.json
 
-  - npm link generator-jhipster
   - npm link generator-jhipster-helloworld
   - jhipster --blueprint helloworld --force --no-insight
 
+  # link again generator-jhipster, because of npm install
+  - cd $HOME/generator-jhipster
+  - npm link
+  - cd $HOME/jhipster-helloworld
+  - npm link generator-jhipster
+  - cd "$APP_FOLDER"/
+  - npm link generator-jhipster
+  
+  # generate entities
   - jhipster entity BankAccount --force --skip-install
   - jhipster entity Label --force --skip-install
   - jhipster entity Operation --force --skip-install
   - jhipster spring-controller Hello --default
   - jhipster spring-service World --default
+  - npm run webpack:build
```

Comment:
without that, mvn test will try to npm install, because of webpack profil, and can't delete node_modules for a strange reason...